x-pack/plugins/apm/server/lib/errors/distribution/get_buckets.ts

```diff
@@ -84,8 +84,5 @@ export async function getBuckets({
       y: bucket.doc_count,
     })
   );
-
-  return {
-    buckets: resp.hits.total.value > 0 ? buckets : [],
-  };
+  return { buckets };
```

Comment:
Thanks for removing this. Glad we don't need this.
@MiriamAparicio this looks similar to the `noHits` occurrences that you are removing in another PR.